frontend/src/components/layout/componentMap.js

```diff
@@ -0,0 +1,13 @@
+import { VIEWERS } from '../../utilities/constants';
+import CphateViewer from '../viewers/CphateViewer';
+import ViewerPlaceholder from '../ViewerPlaceholder';
+import MorphologyViewer from '../viewers/MorphologyViewer';
+
+const componentMap = {
+  [VIEWERS.MorphologyViewer]: MorphologyViewer,
+  [VIEWERS.CphateViewer]: CphateViewer,
+  placeholderViewer: ViewerPlaceholder,
```

Comment:
I don't think we need the placeholderViewer here, we either show the flexLayout (layoutManager.getComponent()) or the ViewerPlaceholder, so the placeholder is not part of the flexLayout.

And we show the ViewerPlaceholder if we have no viewers loaded.